vm/install_elasticsearch.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+# add Elasticsearch Library
+sudo sed -i -e 's/us.archive.ubuntu.com/archive.ubuntu.com/g' /etc/apt/sources.list
+sudo apt-get update
+
+# install OpenJDK7
+sudo apt-get purge openjdk*
```

Comment:
This looks dangerous. Is it really required to uninstall all JDK versions?
It looks like it could (==will) impact other JVM-based DBs.